README.md

```diff
@@ -223,6 +223,13 @@ Supported domains: BLAS, LAPACK, RNG
             <td align="center">Dynamic, Static</td>
             <td align="center">LLVM*, hipSYCL</td>
         </tr>
+        <tr>
+            <td rowspan=1 align="center">DFT</td>
+            <td align="center">Intel GPU</td>
+            <td rowspan=1 align="center">Intel(R) oneAPI Math Kernel Library</td>
+            <td align="center">Dynamic, Static</td>
+            <td align="center">DPC++</td>
+        </tr>
```

Comment:
We should add that `x86 CPU` is supported here.

Reply:
I think that should probably wait until the backend has been added. This should be addressed in this PR https://github.com/oneapi-src/oneMKL/pull/265